include/ua_server.h

```diff
@@ -1056,6 +1056,30 @@ UA_Server_deleteReference(UA_Server *server, const UA_NodeId sourceNodeId,
                           const UA_ExpandedNodeId targetNodeId,
                           UA_Boolean deleteBidirectional);
 
+/**
+ * .. _events:
+ *
+ * Events
+ * ------
+ * The method ``UA_Server_createEvent`` creates an event and represents it as node. The node receives a unique `EventId`
+ * which is automatically added to the node.
+ * The method returns a `NodeId` to the object node which represents the event. The `NodeId` can be used to set the
+ * attributes of the event.
+ *
+ * The method ``UA_Server_triggerEvent`` "triggers" an event by adding it to all monitored items of the specified
+ * origin node and those of all its parents. Any filters specified by the monitored items are automatically applied.
+ * Using this method deletes the node generated by ``UA_Server_createEvent``.
+ */
+#ifdef UA_ENABLE_EVENTS
+
+UA_StatusCode UA_Server_createEvent(UA_Server *server, UA_NodeId eventType,
+                                    UA_NodeId *outNodeId);
+
+UA_StatusCode UA_Server_triggerEvent(UA_Server *server, UA_NodeId *event,
```

Comment:
Other public API methods use const UA_NodeId (not a pointer to nodeid) where possible.
So we can use UA_NODEID_NUMERIC(..) inline in the method call.